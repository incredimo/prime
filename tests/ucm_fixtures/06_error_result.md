```ucm run#r3 lang="python" timeout="1ms"
print("too slow")
```

```ucm result#r3 for="r3"
{"status":"error","error":{"code":"EXEC_TIMEOUT","message":"Command timed out"},"cache_key":"sha256:z","cache_hit":false}