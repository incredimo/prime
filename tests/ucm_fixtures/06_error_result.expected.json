{
  "pair": "r3",
  "status": "error",
  "error.code": "EXEC_TIMEOUT"
}